pagopa/io-app
# fix: [IOCOM-645,IABT-1485] Fix an Android crash when dealing with calendar events

## Short description
This PR prevents a crash on Android devices when trying to add/remove calendar events
![7120a04e-58a1-4ebd-b139-3097892f7d50](https://github.com/pagopa/io-app/assets/5150343/165cbd32-ce05-4032-bc08-ff73afcfe739)

## List of changes proposed in this pull request
- The library used to handle calendar events, `react-native-calendar-events`, has been patched in order to handle event Ids of type `long` (where the reported crash clearly showed that it was trying to convert a >4G numeric string into a signed int32

## How to test
Add and remove calendar events on an Android device and check that there are no problems.
Also check that the patched file matches the same edits [in this PR](https://github.com/wmcmahan/react-native-calendar-events/pull/359/files).
